mitosheet/mitosheet/enterprise/mito_config.py

```diff
@@ -17,6 +17,7 @@
 MITO_CONFIG_CODE_SNIPPETS_VERSION  = 'MITO_CONFIG_CODE_SNIPPETS_VERSION' 
 MITO_CONFIG_CODE_SNIPPETS_URL = 'MITO_CONFIG_CODE_SNIPPETS_URL'
 MITO_CONFIG_DISABLE_TOURS = 'MITO_CONFIG_DISABLE_TOURS'
+MITO_CONFIG_FEATURE_ENABLE_SNOWFLAKE_IMPORT = 'MITO_CONFIG_FEATURE_ENABLE_SNOWFLAKE_IMPORT'
```

Comment:
I think we could refactor this file. Currently, adding a single boolean flag takes adding it in 3-5 places, depending on how you could. Ideally, we'd do it in less than that...